joblib/parallel.py

```diff
@@ -152,10 +152,20 @@ class parallel_backend(object):
     multiprocessing and loky may not be available, in which case joblib
     defaults to threading.
 
-    In addition, if the `dask` and `distributed` Python packages are installed,
-    it is possible to use the 'dask' backend for better scheduling of nested
-    parallel calls without over-subscription and potentially distribute
-    parallel calls over a networked cluster of several hosts.
+    You can also use the `Dask <https://docs.dask.org/en/stable/>`_ joblib
+    backend to distribute work across machines. This works well with
+    scikit-learn estimators with the ``n_jobs`` parameter, for example::
+
+    >>> import joblib  # doctest: +SKIP
+    >>> from sklearn.model_selection import GridSearchCV  # doctest: +SKIP
+    >>> from dask.distributed import Client # doctest: +SKIP
+
+    >>> # create a local Dask cluster
+    >>> client = Client()  # doctest: +SKIP
```

Comment:
Just curious, is there any reason to use a `Client` here and `LocalCluster` + `Client` in your example above?

Reply:
Only because I was trying to keep the docstring example more concise. Also happy to make these consistent if you prefer.

Reply:
I would add the `LocalCluster` so it is clear for users what to change for distributed computation with a real cluster?